src/axom/quest/util/mesh_helpers.hpp

```diff
@@ -55,9 +61,99 @@ mfem::Mesh* make_cartesian_mfem_mesh_3D(const primal::BoundingBox<double, 3>& bb
+#endif
+
+#if defined(AXOM_USE_SIDRE)
+/*!
+  /brief Creates a 3D Cartesian mfem mesh lying within a given bounding box
+
+  \param meshGrp Put the mesh in this Group
+  \param bbox The bounding box for the mesh
+  \param res The resolution of the mesh
+  \param topologyName Name of the blueprint topoloyy to use.
+  \param coordsetName Name of the blueprint coordset to use.
+  \param runtimePolicy Runtime policy, see axom::runtime_policy.
+         Memory in \c meshGrp must be compatible with the
+         specified policy.
+
+  \return The meshGrp pointer
+*/
+axom::sidre::Group* make_structured_blueprint_box_mesh(
+  axom::sidre::Group* meshGrp,
+  const primal::BoundingBox<double, 3>& bbox,
+  const primal::NumericArray<int, 3>& res,
+  const std::string& topologyName = "mesh",
+  const std::string& coordsetName = "coords",
+  axom::runtime_policy::Policy runtimePolicy = axom::runtime_policy::Policy::seq);
+
+axom::sidre::Group* make_unstructured_blueprint_box_mesh(
+  axom::sidre::Group* meshGrp,
+  const primal::BoundingBox<double, 3>& bbox,
+  const primal::NumericArray<int, 3>& res,
+  const std::string& topologyName = "mesh",
+  const std::string& coordsetName = "coords",
+  axom::runtime_policy::Policy runtimePolicy = axom::runtime_policy::Policy::seq);
+
+/*!
+  \brief Convert a structured explicit blueprint mesh to unstructured.
+  \param meshGrp Put the mesh in this Group
+  \param topologyName Name of the blueprint topoloyy to use.
+  \param runtimePolicy Runtime policy, see axom::runtime_policy.
+         Memory in \c meshGrp must be compatible with the
+         specified policy.
+
+  All input mesh data are expected to have the allocator id of
+  meshGrp->getDefaultAllocatorID().  On output, they will also have
+  the same allocator id, even if the intermediate steps have to
+  transfers memory to and from another space.
+*/
+void convert_blueprint_structured_explicit_to_unstructured(
+  axom::sidre::Group* meshGrp,
+  const std::string& topoName,
+  axom::runtime_policy::Policy runtimePolicy);
+
+template <typename ExecSpace>
+void convert_blueprint_structured_explicit_to_unstructured_impl(
+  axom::sidre::Group* meshGrp,
+  const std::string& topoName);
+
+  #if defined(AXOM_USE_CONDUIT)
+/*!
+  \brief Check if blueprint mesh is valid.
+*/
+bool verifyBlueprintMesh(const axom::sidre::Group* meshGrp, conduit::Node info);
+  #endif
 
 #endif
 
+/*!
+  @brief Fill in structured mesh cartesian coordinates.
+  \param xView Vertex x-values array
+  \param yView Vertex y-values array
+  \param zView Vertex z-values array
+  \param domainBox Physical domain
+*/
+void fill_cartesian_coords_3d(axom::runtime_policy::Policy runtimePolicy,
+                              const primal::BoundingBox<double, 3>& domainBox,
+                              axom::ArrayView<double, 3>& xView,
+                              axom::ArrayView<double, 3>& yView,
+                              axom::ArrayView<double, 3>& zView);
```

Comment:
Add comment for the `runtimePolicy` param.

Reply:
Done.